tests/test_questions.py

```diff
@@ -307,6 +307,27 @@ def test_pricing_question_decimal_place_restriction_is_false_by_default(self):
     def test_pricing_question_decimal_place_restriction_can_be_set_true(self):
         assert self.question(decimal_place_restriction=True).decimal_place_restriction is True
 
+    def test_get_data_with_decimal_place_restriction_expands_decimal_point_to_2dp_if_(self):
+        question = self.question(decimal_place_restriction=True)
+        assert question.get_data({'priceMin': '10.'}) == {'priceMin': '10.00'}
+
+    def test_get_data_with_decimal_place_restriction_expands_1dp_to_2dp(self):
+        question = self.question(decimal_place_restriction=True)
+        assert question.get_data({'priceMin': '10.1'}) == {'priceMin': '10.10'}
+
+    def test_get_data_with_decimal_place_restriction_does_not_edit_price_if_integer(self):
+        question = self.question(decimal_place_restriction=True)
+        assert question.get_data({'priceMin': '10'}) == {'priceMin': '10'}
+
+    def test_get_data_with_decimal_place_restriction_does_not_edit_price_if_more_than_2dp(self):
+        question = self.question(decimal_place_restriction=True)
+        assert question.get_data({'priceMin': '10.123'}) == {'priceMin': '10.123'}
+
+    @pytest.mark.parametrize("price", ["11", "11.", "11.2", "11.23", "11.234"])
+    def test_get_data_does_not_edit_price_if_decimal_restriction_is_false(self, price):
+        question = self.question()
+        assert question.get_data({'priceMin': price}) == {'priceMin': price}
```

Comment:
These tests make me think. Clearly there's a lot of repetition, which is good to take out where possible, but I like how the test names clearly document the logic behind each assertion, which would be lost if we parametrized this lot into one test. However, maybe the data would provide enough detail for the correct conclusion to be drawn?

I wonder.

```

@pytest.mark.parametrize('decimal_place_restriction, submitted_value, anticipated_output_value',
                         (
                             (True,  '10.',    '10.00',),
                             (True,  '10.1',   '10.10',),
                             (True,  '10',     '10',),
                             (True,  '10.123', '10.123',),

                             (False, '11',     '11',),
                             (False, '11.',    '11.',),
                             (False, '11.2',   '11.2',),
                             (False, '11.23',  '11.23',),
                             (False, '11.234', '11.234',),
                         )    
```

Questionable.